pybiopax/biopax/base.py

```diff
@@ -121,28 +141,53 @@ def __init__(self,
         self.data_source = data_source
         self.evidence = evidence
 
+    def __repr__(self) -> str:
+        return (
+            f"Entity(standard_name={self.standard_name}, display_name={self.display_name}, "
+            f"all_names={self.all_names}, participant_of={self.participant_of}, "
+            f"availability={self.availability}, data_source={self.data_source},"
+            f"evidence={self.evidence})"
+        )
+
 
 class Gene(Entity):
     """BioPAX Gene"""
     def __init__(self, organism, **kwargs):
         super().__init__(**kwargs)
         self.organism = organism
 
+    def __repr__(self) -> str:
```

Comment:
probably  remove this (and other custom reprs) before merge

Reply:
I think the repr I implemented in a higher-level class that just provides the class name and the standard name could remain, but I agree these detailed reprs could be reverted.